### PR TITLE
Change ngrok to Cloudfalre tunnel 

### PR DESCRIPTION
**Pull request by rashboldb**
### 🛠 Changes being made

Got rid of ngrok and use Cloudflare tunnel instead in order to expose locally running container to the Internet.
Note: this PR is based on top of https://github.com/COXIT-CO/RecruitFlowAI/pull/35 and should be landed after it.

### 🧪 How was it tested? / Demo

Tested locally, i.e. the bot is running as it used to but now we don't have to do ngrok magic.
### 🏎 Quality check

- [ ]  Set base branch to main.
- [x]  Pull request is prepared: Added to sprint, added yourself as assignee, reviewers are added.
- [x]  PR is not very big (about 200 lines of change ideally).
- [x]  I have added comments to my code, particularly in hard-to-understand areas.
- [x]  Doesn't create any new FIXME or TODO comments. If not, please explain.
- [x]  Tested manually (explained the scenario, added screenshots)
- [x]  I have tested my changes locally before pushing them to this branch.
- [x]  I have updated the documentation accordingly.
- [x]  Followed best practice rules:https://google.github.io/styleguide/pyguide.html

### CODE REVIEW

File: docker-compose.yml
Comment (iramykytyn) [inline docker-compose.yml:34]: misstype

**Review by iramykytyn — COMMENTED** [review]
Please convert it to a draft until branch `docker-compose` is merged or exclude changed from other branch from this PR. I can not review it when it's combination of code from 2 branches.
Also always attach recording of fixed issue and explanation how was it tested so I can quickly double check it and run the test.

File: docker-compose.yml
Comment (rashboldb) [reply docker-compose.yml:34]: can't find

**Comment by rashboldb** [comment]
So I've run it over the night, the container is up for more than 20 hours and the bot is working, all good.
I'll remove the rotation code and if all good will open the PR.

**Review by iramykytyn — COMMENTED** [review]
How was it tested? Were you able to run it overnight without the need to restart?

**Comment by iramykytyn** [comment]
> How was it tested? Were you able to run it overnight without the need to restart?

Just saw your comment earlier on it. 
Sounds good. I will also try to run it until tomorrow and approve it if it works for me as well!

**Review by iramykytyn — APPROVED** [review]

**Review by iramykytyn — APPROVED** [review]
Run overnight, everything works perfect! Good job!

**Pull request merged**
